pkgs/tools/networking/goflow2/1.x.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "goflow2";
+  version = "1.3.6";
+
+  src = fetchFromGitHub {
+    owner = "netsampler";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-/SAh8naChamgNR2haoW6FC+dHTP9iAZpyk3p2qQ4/PI=";
+  };
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X=main.version=${version}"
+  ];
+
+  vendorHash = "sha256-jWm/061alLKTzn53uKun1qj2TM77pjltQlZl1/dTd80=";
+
+  meta = with lib; {
```

Comment:
`lib.getExe` in the module complains that `meta.mainProgram` is not set here.

Reply:
fixed